src/commands/api-mesh/source/__fixtures__/0.0.1-test-01.json

```diff
@@ -0,0 +1,54 @@
+{
+	"name": "Test 01",
+	"version": "0.0.1",
+	"description": "Mock for valid connector",
+	"author": "VZ",
+	"provider": {
+		"name": "Commerce",
+		"handler": {
+			"graphql": {
+				"endpoint": "https://venia.magento.com/graphql/"
+			}
+		},
+		"transforms": [
+			{
+				"rename": {
+					"mode": "bare | wrap",
+					"renames": [
+						{
+							"from": {
+								"type": "Query",
+								"field": "compareList"
+							},
+							"to": {
+								"type": "Query",
+								"field": "productCompareList"
+							}
+						}
+					]
+				}
+			},
+			{
+				"filterSchema": {
+					"mode": "bare | wrap",
+					"filters": [
+						"Query.!category",
+						"Query.!customerOrders",
+						"Query.!urlResolver",
+						"Query.!wishlist"
+					]
+				}
+			},
+			{
+				"cache": [
```

Comment:
I understand this is just a test but we don't support cache transform at the moment. If this is submitted to SMS, it will fail with an appropriate error.

Reply:
Removed